actual/database.py

```diff
@@ -571,7 +574,14 @@ class ZeroBudgets(SQLModel, table=True):
 
     id: Optional[str] = Field(default=None, sa_column=Column("id", Text, primary_key=True))
     month: Optional[int] = Field(default=None, sa_column=Column("month", Integer))
-    category: Optional[str] = Field(default=None, sa_column=Column("category", Text))
+    category: Optional[str] = Field(default=None, sa_column=Column("category", ForeignKey("categories.id")))
```

Comment:
Probably best to rename this to `category_id` and include the relationship `category` after all column definitions, to separate the relationship from the columns.